src/main/webapp/stylesheets/teammatesCommon.css

```diff
@@ -1184,14 +1184,33 @@ Prevent Long URL From Breaking Out of Container
     margin-right: 5px;
 }
 
-/*
-Glyphicon for success status messages
-*/
+/*//////////////////////////////////////////////////////////
+////////////////////// Status Icons //////////////////////
+//////////////////////////////////////////////////////////*/
 
-.icon-success:before {
+.icon-success:before,
+.icon-warning:before,
+.icon-danger:before,
+.icon-info:before {
+    /*noinspection CssNoGenericFontName*/
```

Comment:
@LiHaoTan I'm not sure we should add a suppression just to turn off a warning that is very specific to IntelliJ. Doing this implies that we regard the default inspection settings of a single IDE as authoritative, and always to be followed whenever we write code. This is outside the spirit that our project is IDE-agnostic. Only general warnings that apply to most IDEs should be considered.